File: baselib/src/main/java/com/stjy/baselib/net/request/BaseApiResult.java
package com.stjy.baselib.net.request;

import com.google.gson.annotations.SerializedName;
import com.zhouyou.http.model.ApiResult;

/**
 * @author acb
 */
public class BaseApiResult<T> extends ApiResult<T> {

    @SerializedName("success")
    private boolean success;

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    @Override
    public boolean isOk() {
        return "200".equals(getCode());
    }

}

File: baselib/src/main/java/com/stjy/baselib/net/net1/request/BaseApiResult.kt
package com.stjy.baselib.net.net1.request

import com.zhouyou.http.model.ApiResult

/**
 * @author acb
 */
class BaseApiResult<T> : ApiResult<T>() {
    override fun isOk(): Boolean {
        return code==0
    }
}